source/draw/RasterMonochrome.ooc

```diff
@@ -79,7 +114,34 @@ RasterMonochrome: class extends RasterPacked {
 		dimensions := endDst - startDst
 		endSrc := startSrc + dimensions
 		if (startSrc x >= 0 && startSrc y >= 0 && endSrc x <= source size x && endSrc y <= source size y && startDst x >= 0 && startDst y >= 0 && endDst x <= this size x && endDst y <= this size y) {
-			this _blitWhiteRaw(this buffer pointer, (source as This) buffer pointer, startSrc x, startSrc y, startDst x, startDst y, dimensions x, dimensions y, this stride, (source as This) stride)
+			this _blitRaw(this buffer pointer, (source as This) buffer pointer, startSrc x, startSrc y, startDst x, startDst y, dimensions x, dimensions y, this stride, (source as This) stride, blendMode)
+		}
+	}
+	exactDistance: func (other: This) -> Int {
+		if (this size != other size)
+			Int maximumValue
+		else {
+			result := 0
+			thisUpperLeft := this buffer pointer
+			otherUpperLeft := other buffer pointer
+			thisStride := this stride
+			otherStride := other stride
+			thisLineStart := thisUpperLeft
+			otherLineStart := otherUpperLeft
+			for (y in 0 .. this size y) {
+				thisPixel := thisLineStart
+				otherPixel := otherLineStart
+				for (x in 0 .. this size x) {
```

Comment:
I can probably optimize it all the way here too but Emil might dislike that since this comparison function is only executed for regression tests. I just copied the render loop to save some time on regression tests.